tests/integration/helpers.bash

```diff
@@ -13,19 +13,19 @@ GOPATH="$(mktemp -d --tmpdir runc-integration-gopath.XXXXXX)"
 TESTDATA="${INTEGRATION_ROOT}/testdata"
 
 # Busybox image
-BUSYBOX_IMAGE="$BATS_TMPDIR/busybox.tar"
-BUSYBOX_BUNDLE="$BATS_TMPDIR/busyboxtest"
+BUSYBOX_IMAGE="$BATS_RUN_TMPDIR/busybox.tar"
+BUSYBOX_BUNDLE="$BATS_RUN_TMPDIR/busyboxtest"
 
 # hello-world in tar format
 HELLO_FILE=$(get_hello)
 HELLO_IMAGE="$TESTDATA/$HELLO_FILE"
-HELLO_BUNDLE="$BATS_TMPDIR/hello-world"
+HELLO_BUNDLE="$BATS_RUN_TMPDIR/hello-world"
 
 # debian image
-DEBIAN_BUNDLE="$BATS_TMPDIR/debiantest"
+DEBIAN_BUNDLE="$BATS_RUN_TMPDIR/debiantest"
 
 # CRIU PATH
-CRIU="$(which criu 2>/dev/null || true)"
+CRIU="$(command -v criu || true)"
```

Comment:
OK I guess this enabled checkpoint/restore tests on CentOS 7, and the lazy migration one failed with unclear diagnostics. Looking.

Reply:
Moved this commit into https://github.com/opencontainers/runc/pull/2761 to "fix" the whole PR.

Reply:
Moved the patch to a separate PR #2671.